internal/condition/eval/eval_test.go

```diff
@@ -27,53 +27,56 @@ func TestEvaluateTupleCondition(t *testing.T) {
 		{
 			name:     "condition_in_tuple_key_not_found_in_model",
 			tupleKey: tuple.NewTupleKeyWithCondition("document:1", "viewer", "user:maria", "unknown", nil),
-			model: parser.MustTransformDSLToProto(`model
-	schema 1.1
-type user
-
-type document
-  relations
-    define can_view: [user with correct_ip]
-
-condition correct_ip(ip: string) {
-	ip == "192.168.0.1"
-}`),
+			model: parser.MustTransformDSLToProto(`
+				model
+				schema 1.1
```

Comment:
should schema be indented?

Reply:
Good catch Adrian, based on a [comment](https://github.com/openfga/openfga/pull/1597#discussion_r1597522757) on closed PR, that was desired.

I'll re-run script to incorporate that and re-submit today.

Reply:
Good catch Adrian, based on a [comment](https://github.com/openfga/openfga/pull/1597#discussion_r1597522757) on closed PR, that was desired.

I'll re-run script to incorporate that and re-submit today.